tests/encryption/test_intransit_encryption_sanity.py

```diff
@@ -0,0 +1,64 @@
+import logging
+import pytest
+
+from ocs_ci.ocs.resources.storage_cluster import (
+    in_transit_encryption_verification,
+    set_in_transit_encryption,
+    get_in_transit_encryption_config_state,
+)
+from ocs_ci.framework.pytest_customization.marks import (
+    tier1,
+    skipif_ocs_version,
+)
+from ocs_ci.framework import config
+
+log = logging.getLogger(__name__)
+
+
+class TestInTransitEncryptionSanity:
+    @pytest.fixture(autouse=True)
+    def set_encryption_at_teardown(self, request):
+        def teardown():
+            if config.ENV_DATA.get("in_transit_encryption"):
+                set_in_transit_encryption()
+            else:
+                set_in_transit_encryption(enabled=False)
+
+        request.addfinalizer(teardown)
+
+    @tier1
+    @skipif_ocs_version("<4.13")
+    @pytest.mark.polarion_id("OCS-4861")
+    def test_intransit_encryption_enable_disable_statetransition(self):
+        """
+        The test does the following:
+        1. Enable in-transit Encryption if not Enabled.
+        2. Verify in-transit Encryption is Enable on setup.
+        3. Disable Encryption
+        4. Verify in-transit encryption configuration is removed.
+        5. Enable encryption Again and verify it.
+        6. Verify in-transit encryption config is exists.
+
+        """
+        if not get_in_transit_encryption_config_state():
```

Comment:
We need to know here if it is deployment config, and in-transit encryption is not enabled we should fail the test, else enable in-transit encryption.

Reply:
Done